compose.dev.yml

```
@@ -87,8 +87,10 @@ services:
         - LDAP_ADMIN_USERNAME=admin
         - LDAP_ADMIN_PASSWORD=secret
         - BITNAMI_DEBUG=true
+        - LDAP_EXTRA_SCHEMAS=cosine,inetorgperson,custom
     volumes:
         - ./ldap:/ldifs:z
+        - ./ldap/schemas/custom.ldif:/opt/bitnami/openldap/etc/schema/custom.ldif:z
```

On peut légèrement simplifier ça parce que l’image charge automatiquement le schema custom s’il se trouve dans le répertoire `/schema`. Et par contre pour éviter de charger le schema sous `/ldifs`, je changerais aussi ça :


```suggestion
    volumes:
        - ./ldap/tree.ldif:/ldifs/tree.ldif:z
        - ./ldap/schemas/custom.ldif:/schema/custom.ldif:z
```